projects/cashmere/src/lib/modal/modal-window.component.ts

```diff
@@ -6,7 +6,11 @@ import {ActiveModal} from './active-modal';
 @Component({
     selector: 'hc-modal-window',
     template: `
-        <div [class]="'hc-modal hc-modal-' + _size"><ng-content></ng-content></div>
+        <hc-modal-drag [disableDrag]="_isDraggable">
```

Comment:
@corykon - I want to talk this template over with you before a merge.  I have a feeling there is a more elegant way to do this that doesn't include the `hc-modal-drag` tag at all if `_isDraggable` is false.  The tough part is that it's the parent, so it's tricky to use `ngIf` here.  Also because if you put two `ng-content` blocks in an if/else setup, Angular doesn't know which one to actually put the content in. So what we've got here works - I don't love the extra tag where it isn't needed.  If you've got any Angular tricks up your sleeve, let me know.

Reply:
No angular tricks coming to mind... I think my preferred approach would have been to avoid the `hc-modal-drag` component completely, and just directly use cdkDrag in this template, maybe something like this:
```
<div [class]="'hc-modal hc-modal-' + _size" cdkDrag [cdkDragDisabled]="!_isDraggable">
     <span [hidden]="!_isDraggable" class="hc-modal-drag-handle></span>
     <ng-content></ng-content>
</div>
```

Reply:
Ya know - while I was doing the cleanup I didn't even take a step back to consider revising to do it without the component, I bet that wouldn't be too hard to change.  I'll give it a look this afternoon.